crates/solvers/src/domain/solver.rs

```diff
@@ -35,6 +36,7 @@ pub struct Config {
     pub max_partial_attempts: usize,
     pub solution_gas_offset: eth::SignedGas,
     pub native_token_price_estimation_amount: eth::U256,
+    pub node_url: Option<Url>,
```

Comment:
We are naming `node_url` the config (which is a generic name) but later we assume this is `uni_v3_quoter_v2` URL. So maybe it would be more readable to use the same naming in both places?